docs/features/details/Details.md

```diff
@@ -29,47 +27,58 @@ An object's further detail properties as entries with
                  e.g. a simple string or a more complex object
 end note
 
+Branch --> Details : details
+Build --> Details : details
+UseCase --> Details : details
+Scenario --> Details : details
+Step --> Details : details
+
+class Object
+
+Details --> "*" Object
+
+
 class String
 
+Object <|-- String
+
+
 class ObjectDescription {
     name: String
     type: String
     details: Details
 }
 
+Object <|-- ObjectDescription
+
```

Comment:
The association from _ObjectDescription_ to _Details_ is missing.
Or is it omitted on purpose so the diagram isn't too cluttered?

Reply:
th elast sentence is true for your other two comments ;-) But for this one I am not sure ;-)

Reply:
yes., here as well

Reply:
we decided to ommit on purpose